planner/core/casetest/physicalplantest/testdata/plan_suite_out.json

```diff
@@ -2099,7 +2099,7 @@
       },
       {
         "SQL": "select * from t where t.c = 1 and t.e = 1 order by t.d limit 1",
-        "Best": "IndexLookUp(Index(t.c_d_e)[[1,1]]->Sel([eq(test.t.e, 1)])->Limit, Table(t))"
```

Comment:
The change seems strange. Pushing down Limit seems always better than not pushing down?

Reply:
I debug this locally and find that if we push the `Limit` down, there can be an extra `Proj` in the intermediate plan, which increases the cost of the limit-push-down plan. 
This extra `Proj` finally will be removed by `postOptimization` operation, so we cannot see it in the final Plan.
This issue is a little hard to fix under our current physical optimization implementation, and it can be resolved by `tidb_opt_limit_push_down_threshold`, so how about ignoring this issue now?
<img width="1227" alt="image" src="https://github.com/pingcap/tidb/assets/7499936/76ae55fa-6b01-49bf-96e4-62fb3ee195eb">

Reply:
I think it's ok.